packages/net/tcp_connection.pony

```diff
@@ -381,7 +415,12 @@ actor TCPConnection
 
             // Don't call _complete_writes, as Windows will see this as a
             // closed connection.
-            _pending_writes()
+            ifdef not windows then
+              if _pending_writes() then
+                //sent all data; release backpressure
```

Comment:
The style of this comment isn't quite a match for the surrounding style.  Can you add a space before the first letter, capitalize it, and maybe also add a period at the end?